test/lt.test.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const IlpPacket = require('ilp-packet')
+const chai = require('chai')
+const assert = chai.assert
+const LT = require('..').LT
+const MockPlugin = require('./mocks/mockPlugin')
+const chaiAsPromised = require('chai-as-promised')
+chai.use(chaiAsPromised)
+
+describe('IT', function () {
```

Comment:
LT?

Reply:
Loopback Transport (see https://github.com/interledger/rfcs/pull/416/files#diff-ce1d0a4db7a75b9030d0436e521d092bR115)

Reply:
sorry, I meant that you wrote `IT` there and I think it should be `LT`